src/Plugin/Field/FieldFormatter/StrawberryMapFormatter.php

```diff
@@ -476,10 +477,11 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
           '#type' => 'number',
           '#title' => $this->t('Maximum height'),
           '#default_value' => $this->getSetting('max_height'),
+          '#description' => $this->t('The minimum permitted value is 62 pixels to ensure that the zoom controls are visible on the map.'),
           '#size' => 5,
           '#maxlength' => 5,
           '#field_suffix' => $this->t('pixels'),
-          '#min' => 0,
+          '#min' => 62,
```

Comment:
Ok.. so we already had min 0 here ... so the original use case was solved? Won't fight it

Reply:
Just making it match what we enforce later.